src/components/Auth/AddTodo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+type Props = {
+  newTodoField: React.RefObject<HTMLInputElement>;
```

Comment:
But you don't use this ```ref``` here.
And after adding you don't focus on the input field.
Add the same UseEffect with this ```ref``` as at App.tsx